astroquery/sdss/core.py

```diff
@@ -1055,12 +1076,20 @@ def _args_to_payload(self, coordinates=None, radius=2. * u.arcsec,
                 raise ValueError('must specify at least one of `coordinates`, '
                                  '`run`, `camcol` or `field`')
 
-        sql = "{0} {1} {2} {3}".format(q_select, q_from, q_join, q_where)
-
-        request_payload = dict(cmd=sql, format='csv')
+        sql = "{0} {1} {2}".format(q_select, q_from, q_where)
```

Comment:
```suggestion
        sql = f"{q_select} {q_from} {q_where}"
```

Reply:
Again, minimal change.

Reply:
These lines are being edited anyways and the change is not user-facing.

Reply:
Done